service/service.go

```diff
@@ -79,5 +138,7 @@ func New(config Config) (*Service, error) {
 func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
 		// Insert service startup logic here.
+		// Start the controller.
```

Comment:
Lets remove the `Insert service startup logic here.` comment above. Its not needed anymore.